cores/arduino/USBCore.cpp

```diff
@@ -431,6 +448,14 @@ class ClassCore
         }
 };
 
+void (*oldResetHandler)(usb_dev *usbd);
+void handleReset(usb_dev *usbd)
+{
+    Serial1.print("/");
```

Comment:
Is this a debugging leftover that prints something to the hardware UART?

Reply:
oops! give me a tick to push a fix